src/stores/Frame/Region/RegionStore.ts

```diff
@@ -198,6 +198,10 @@ export class RegionStore {
         }
     }
 
+    /**
+     * Getter for the sizes of regions and annotations
```

Comment:
I suggest we revise it to "Returns the sizes of regions and annotations. For line regions and annotations, vector annotations, and ruler annotations, returns x and y absolute displacements."